Makefile

```diff
@@ -36,17 +46,20 @@ all:
 	@echo "src-link uintent $(ROOT_DIR)/package" >> $(UINTENT_SDK_DIR)/feeds.conf
 	@$(UINTENT_SDK_DIR)/scripts/feeds update uintent
 	@$(UINTENT_SDK_DIR)/scripts/feeds install -a
-	@echo 'CONFIG_UINTENT_CONFIG_DIR="$(ROOT_DIR)/config"' > $(UINTENT_SDK_DIR)/.config
+	@echo 'CONFIG_UINTENT_CONFIG_DIR="$(UINTENT_CONFIG_DIR)"' > $(UINTENT_SDK_DIR)/.config
 	$(SDK_MAKE) defconfig
 	$(SDK_MAKE) package/uintent/clean
 	$(SDK_MAKE) package/uintent-config/clean
 	$(SDK_MAKE) package/uintent/compile
+	$(SDK_MAKE) package/uintent-usteer/clean
+	# TODO: maybe only conditionally compile uintent-usteer
```

Comment:
I would argue that usteer is always useful. A single band AP is an outlier these days, and in any other case we already benefit from band-steering.